DotNet/overrides.js

```diff
@@ -54,4 +54,9 @@
 
     "should use regions padding": {skipEmit: true},
     "should use regions padding with vg": {skipEmit: true},
+
+    'regionId can be specified as part of checkSettings classic': {skipEmit: true},
+    'regionId can be specified as part of checkSettings with vg': {skipEmit: true},
+    'coded regions with selectors or elements should automatically include a regionId classic': {skipEmit: true},
+    'coded regions with selectors or elements should automatically include a regionId with vg': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9690162)
```suggestion
    "coded regions with selectors or elements should automatically include a regionId with vg": {skipEmit: true},
```